File: 2022/week_6/psets/sentimental-mario-less/mario.py
# Prints a simple pyramid of a height
h = 0

# Getting user input
while h > 8 or h < 1:
    try:
        h = int(input("Height: "))
    except ValueError:
        pass

# Printing the pyramid
for i in range(1, h + 1):
    print(" " * (h - i) + "#" * i)

File: 2022/week_1/psets/mario_less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Getting user input
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);


    // Printing the pyramid
    for (int i = 1; i <= height; i++)
    {
        for (int j = height - i; j > 0; j--)  // Printing spaces
        {
            printf(" ");
        }
        for (int j = 1; j <= i; j++)  // Printing hashes
        {
            printf("#");
        }
        printf("\n");  // Printing a newline
    }
}